.github/workflows/ci.yml

```diff
@@ -22,12 +26,16 @@ jobs:
         run: make check
       - name: distcheck
         run: make distcheck
-  test-ubuntu-oldest:
-    runs-on: ubuntu-16.04
+
+  test-centos:
+    runs-on: ubuntu-latest
+    container: centos
     steps:
       - uses: actions/checkout@v2
+      - name: install ddclient dependencies
+        run: dnf install -y perl-version
       - name: install test dependencies
-        run: sudo apt-get install libtest-warnings-perl
+        run: dnf --enablerepo=PowerTools install -y autoconf automake make perl-Test-Warnings
```

Comment:
I'm not familiar with CentOS. Is this PowerTools repo required? I would like to test it on a CentOS install that is as vanilla as possible.

Reply:
``PowerTools - Available only for CentOS8, the PowerTools repository provides most of the developer tools. Disabled by default.`` https://wiki.centos.org/AdditionalResources/Repositories

For ``perl-Test-Warnings`` it is required and is fine to install.

Reply:
I used https://centos.pkgs.org/8/centos-powertools-x86_64/perl-Test-Warnings-0.026-7.el8.noarch.rpm.html to get the required packages and instructions.